packages/darklang/languageTools/parser/identifiers.dark

```diff
@@ -54,14 +54,43 @@ module Darklang =
           (node: ParsedNode)
           : Stdlib.Result.Result<WrittenTypes.QualifiedTypeIdentifier, String> =
           if node.typ == "qualified_type_name" then
+            let filteredChildren =
+              node.children
+              |> Stdlib.List.filter (fun n ->
+                n.typ != "type_args"
+                && n.fieldName != Stdlib.Option.Option.Some "symbol_open_angle"
```

Comment:
not very happy with this solution, any suggestion of alternatives please?

Reply:
I think `type_args` should include the `<` and `>` characters - then, the task seems simpler: make sure that `<` is the first thing in the list, `>` is the last thing in the list, and then extract the other things as you're doing. Then, you just need to look for the one `type_args` node, here.